sdk/lib/crt/wine/rosglue_i386.s

```diff
@@ -0,0 +1,47 @@
+
+#include <asm.inc>
+
+.code
+
+EXTERN _RtlUnwind@16:PROC
+
+// ASM wrapper for Wine code. This is needed, because Wine code expects
+// RtlUnwind to restore the non-volatile registers, before returning, but
+// ours / the native one does not do that.
```

Comment:
"the native one" == Windows' one? Is this true for all Windows versions?

Reply:
It's true for Win 2003 (see WHS test linked in the description) and Windows 10 WOW64.

Reply:
ok, "wow64" because the rtlunwind is x86 only?

Reply:
x64 also has RtlUnwind, but that works differently. The issue here is x86 specific.